nms/app/packages/magmalte/app/views/equipment/GatewayLogs.js

```diff
@@ -202,6 +216,21 @@ export default function GatewayLogs() {
   function LogsFilter() {
     return (
       <Grid container justify="flex-end" alignItems="center" spacing={1}>
+        <Grid item>
+          <FormControlLabel
+            control={
+              <Checkbox
+                checked={isAutorefreshing}
+                onChange={() => setIsAutorefreshing(current => !current)}
+              />
+            }
+            label={
+              <Text variant="body3" className={classes.autorefreshCheckbox}>
+                Update Automatically
```

Comment:
minor nit : "autorefresh" term is crisp, "Update automatically" takes up a lot of place. Also do you have any opinions on this coming below the datetime selector or right of the end date ?

Reply:
@karthiksubraveti , I like it when it's displayed below the datetime selector. What do you think?
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/43514877/93765519-b2120d00-fc1d-11ea-920b-855040bf4016.png">

Reply:
Just saw this now. Now thinking about this. Having a checkbox might not be ideal. I feel having a small refresh button with a menu containing the various autorefresh time options might be better. This can be done later. This is good for now.